docs/UserGuide.md

```diff
@@ -108,7 +111,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
-
+*  `edit 1 ps/completed py/paid cs/potential` Edits the project status, payment status and client status of the 1st person to be `completed`, `paid` and `potential` respectively.
```

Comment:
Small nit, would it be better to have the payment status be unpaid as they're a potential client?